test/resources/code/scala/hello-scala-spark/hello-scala-spark.sbt

```diff
@@ -1,9 +1,9 @@
 name := "hello-scala-spark"
 version := "1.0"
-scalaVersion := "2.11.12"
+scalaVersion := "2.12.12"
 useCoursier := false
 retrieveManaged := true
-libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.4.4"
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "3.0.0"
```

Comment:
should this be 3.0 or 3.1 ? As I saw our latest release will be using spark 3.1

Reply:
changed, waiting for test code build job to pass